### PR TITLE
Gamedb : add various fixes for various games and remove not needed patches 

### PR DESCRIPTION
**Pull request by Mrlinkwii**
### Description of Changes

Adds various patches which fixes games 
### Rationale behind Changes

fixes said games 
### Suggested Testing Steps

look at the  CI and test said games

### CODE REVIEW

File: bin/GameIndex.yaml [hunk 1]
```diff
@@ -8227,6 +8218,8 @@ SLES-50538:
 SLES-50539:
   name: "James Bond 007 - Agent Under Fire"
   region: "PAL-M6"
+  clampModes:
+    vuClampMode: 2 #Fixes currupt textures.
```
Comment (RedDevilus) [inline bin/GameIndex.yaml:8222]:
```suggestion
    vuClampMode: 2 # Fixes currupt textures. 
```

File: bin/GameIndex.yaml [hunk 2]
```diff
@@ -25153,8 +25132,15 @@ SLPM-66160:
   roundModes:
     eeRoundMode: 0
 SLPM-66163:
-  name: "Fuuuraiki 2"
+  name: "Fuuraiki 2"
   region: "NTSC-J"
+  patches:
+    FDF6D8C2:
+      content: |-
+        author=prafull
```
Comment (RedDevilus) [inline bin/GameIndex.yaml:25140]:
```suggestion
        author=Prafull
```

File: bin/GameIndex.yaml [hunk 3]
```diff
@@ -26099,8 +26085,15 @@ SLPM-66457:
         // This patch skips over the stack code, allowing the game to boot.
         patch=1,EE,001579D8,word,10000003
 SLPM-66458:
-  name: "Acheter Fuuuraiki"
+  name: "Fuuraiki"
   region: "NTSC-J"
+  patches:
+    37CA944E:
+      content: |-
+        author=prafull
```
Comment (RedDevilus) [inline bin/GameIndex.yaml:26093]:
```suggestion
        author=Prafull
```

File: bin/GameIndex.yaml [hunk 4]
```diff
@@ -25153,8 +25132,15 @@ SLPM-66160:
...
     eeRoundMode: 0
 SLPM-66163:
-  name: "Fuuuraiki 2"
+  name: "Fuuraiki 2"
   region: "NTSC-J"
+  patches:
+    FDF6D8C2:
+      content: |-
+        author=prafull
+        //Skip sceipusync
```
Comment (RedDevilus) [inline bin/GameIndex.yaml:25141]: // with spaces everywhere too

File: bin/GameIndex.yaml [hunk 5]
```diff
@@ -27797,6 +27797,12 @@ SLPM-67502:
   name: "Devil May Cry"
   region: "NTSC-K"
   compat: 5
+SLPM-67505:
+  name: "James Bond 007 - Agent Under Fire"
+  region: "NTSC-K"
+  compat: 5
+  clampModes:
+    vuClampMode: 2 #Fixes currupt textures.
```
Comment (RedDevilus) [inline bin/GameIndex.yaml:27805]:
```suggestion
    vuClampMode: 2 # Fixes currupt textures.
```

File: bin/GameIndex.yaml [hunk 6]
```diff
@@ -33626,6 +33632,8 @@ SLUS-20265:
   name: "James Bond 007 - Agent Under Fire"
   region: "NTSC-U"
   compat: 5
+  clampModes:
+    vuClampMode: 2 #Fixes currupt textures.
```
Comment (RedDevilus) [inline bin/GameIndex.yaml:33636]:
```suggestion
    vuClampMode: 2 # Fixes currupt textures. 
```

**Review by RedDevilus — CHANGES_REQUESTED** [review]
Some minor stuff you could adjust.

File: bin/GameIndex.yaml [hunk 7]
```diff
@@ -8227,6 +8218,8 @@ SLES-50538:
 SLES-50539:
   name: "James Bond 007 - Agent Under Fire"
   region: "PAL-M6"
+  clampModes:
+    vuClampMode: 2 # Fixes currupt textures.
```
Comment (Rumi-Larry) [inline bin/GameIndex.yaml:8222]: typo: "corrupt"

File: bin/GameIndex.yaml [hunk 8]
```diff
@@ -27797,6 +27797,12 @@ SLPM-67502:
   name: "Devil May Cry"
   region: "NTSC-K"
   compat: 5
+SLPM-67505:
+  name: "James Bond 007 - Agent Under Fire"
+  region: "NTSC-K"
+  compat: 5
+  clampModes:
+    vuClampMode: 2 # Fixes currupt textures.
```
Comment (Rumi-Larry) [inline bin/GameIndex.yaml:27805]: ditto

File: bin/GameIndex.yaml [hunk 9]
```diff
@@ -33626,6 +33632,8 @@ SLUS-20265:
   name: "James Bond 007 - Agent Under Fire"
   region: "NTSC-U"
   compat: 5
+  clampModes:
+    vuClampMode: 2 # Fixes currupt textures.
```
Comment (Rumi-Larry) [inline bin/GameIndex.yaml:33636]: ditto

**Comment by RedDevilus** [comment]
One last improvement is to change
Disc1 and Disc2 entries in the serial name to

[Disc1of2]

[Disc2of2]

It's consistent with the other entries in the GameDB

**Comment by RedDevilus** [comment]
At least Onimusha will look less like the deep-fried meme in the menus.

File: bin/GameIndex.yaml [hunk 10]
```diff
@@ -18667,25 +18620,53 @@ SLES-82037:
     - "SLES-82036"
     - "SLES-82037"
 SLES-82038:
-  name: "Onimusha - Dawn of Dreams [Disc1]"
+  name: "Onimusha - Dawn of Dreams [Disc1 of 2]"
```
Comment (RedDevilus) [inline bin/GameIndex.yaml:18623]:
```suggestion
  name: "Onimusha - Dawn of Dreams [Disc1of2]"
```

File: bin/GameIndex.yaml [hunk 11]
```diff
@@ -18667,25 +18620,53 @@ SLES-82037:
...
+        // Disable print - we need that loop!
+        // Without loop ipu break itself.
+        patch=1,EE,0010FDC8,word,34190180
+        patch=1,EE,0010FDCC,word,1720FFFF
+        patch=1,EE,0010FDD0,word,2739FFFF
+        patch=1,EE,0010FDD4,word,03E00008
+        patch=1,EE,0010FDD8,word,00000000
 SLES-82039:
-  name: "Onimusha - Dawn of Dreams [Disc2]"
+  name: "Onimusha - Dawn of Dreams [Disc2 of 2]"
```
Comment (RedDevilus) [inline bin/GameIndex.yaml:18646]:
```suggestion
  name: "Onimusha - Dawn of Dreams [Disc2of2]"
```

**Review by RedDevilus — CHANGES_REQUESTED** [review]
Either we do spaces everywhere or remove spaces for saying which disc it is.

**Review by RedDevilus — APPROVED** [review]

**Review by JordanTheToaster — APPROVED** [review]

**Pull request merged**